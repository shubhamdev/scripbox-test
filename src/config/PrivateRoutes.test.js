import React from "react";
import { unmountComponentAtNode } from "react-dom";
import { shallow } from "enzyme";
import PrivateRoutes from "./PrivateRoutes";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe("PrivateRoutes", () => {
  it('should render PrivateRoutes correctly in "debug" mode', () => {
    const component = shallow(<PrivateRoutes debug />);
    expect(component).toMatchSnapshot();
  });
  it("should render PrivateRoutes correctly with no props", () => {
    const component = shallow(<PrivateRoutes />);
    expect(component).toMatchSnapshot();
  });
});
