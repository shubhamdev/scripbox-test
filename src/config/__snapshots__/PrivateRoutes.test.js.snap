// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoutes should render PrivateRoutes correctly in "debug" mode 1`] = `
<Route
  debug={true}
  render={[Function]}
/>
`;

exports[`PrivateRoutes should render PrivateRoutes correctly with no props 1`] = `
<Route
  render={[Function]}
/>
`;
